en/api/configuration-hooks.md

```diff
@@ -36,3 +36,116 @@ Internally, hooks follow a naming pattern using colons (e.g., `build:done`). For
 - [ModulesContainer hooks](https://nuxtjs.org/api/internals-module-container#hooks)
 - [Builder hooks](https://nuxtjs.org/api/internals-builder#hooks)
 - [Generator hooks](https://nuxtjs.org/api/internals-generator#hooks)
+
+## Examples
+
+### Redirect to router.base when not on root
+
+Let´s say you want to serve pages as `/portal` instead of `/`.
+
+This is maybe an edge-case, and the point of _nuxt.config.js_’ `router.base` is for when a Web server will serve Nuxt elsewhere than the domain root.
+
+But when in local development, hitting _localhost_, when router.base is not / returns a 404.
+In order to prevent this, you can setup a Hook.
+
+Maybe redirecting is not the best use-case for a production Web site, but this will help you leverage Hooks.
+
+To begin, you [can change `router.base`](/api/configuration-router#base); Update your `nuxt.config.js`:
+
+```js
+// nuxt.config.js
+import hooks from './hooks'
+export default {
```

Comment:
Better to keep consistent code style with other pages which has no comma after last property.

Reply:
Done!